tfm/tfm.spec

```diff
@@ -6,10 +6,10 @@
 %{?scl_package:%scl_package %scl}
 
 # Fallback to sclo-ror42 etc. when scldevel's not in the buildroot
-%{!?scl_ror:%global scl_ror sclo-ror42}
-%{!?scl_prefix_ror:%global scl_prefix_ror %{scl_ror}-}
-%{!?scl_ruby:%global scl_ruby rh-ruby22}
-%{!?scl_prefix_ruby:%global scl_prefix_ruby %{scl_ruby}-}
+%global scl_ror tfm-ror51
+%global scl_prefix_ror %{scl_ror}-
+%global scl_ruby rh-ruby24
+%global scl_prefix_ruby %{scl_ruby}-
```

Comment:
I modified this to make it explicit. I was not a big fan of the fact that these would get magically set by the buildroot meaning that the buildroot could be different than the defaults specified here. This in turn caused this to explicitly fail since it forced it to require tfm-ror51 and rh-ruby24 in a previous run. Now that I have modified the buildroot, this should pass accurately.